### PR TITLE
py-pytest-env: new port, version 1.1.3

### PR DESCRIPTION
**Pull request by nilason**
#### Description

New `py{*}-pytest-env` port, version 1.1.3.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.2.1 23C71 arm64
Xcode 15.1 15C65

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by pmetzger** [comment]
@reneeotten I defer to you on all things Python...

File: python/py-pytest-env/Portfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                    port:py${python.version}-pytest
+
+    if {${python.version} < 311} {
+        depends_run-append \
+                    port:py${python.version}-tomli
+    }
+
+    pre-test {
+        test.env    PYTHONPATH=[glob -nocomplain ${destroot}/${python.pkgd}]
+    }
```
Comment (reneeotten) [inline python/py-pytest-env/Portfile:45]: is this "pre-test" needed - presumably the "python" PortGroup sets this already to an appropriate value.

Comment (nilason) [reply python/py-pytest-env/Portfile]: Déjà vu. You have rightfully given this note before. Then, and now, at some point _needed_ to add the PYTHONPATH for the test to work. Testing this again it works without. Wonder how that came to be?
I have now removed the lines.

**Comment by pmetzger** [comment]
Looks like this should be done now presuming that @reneeotten is pleased.

**Pull request merged**